### DISCUSSION TITLE
RuntimeUpdatesProcessor -> disabling Live reload

### DISCUSSION THREAD

**Discussion by michal-duchnowski** [quarkusio/quarkus#38300] [Q&A]
Hi everyone.
I've performance problem with Live reload functionality.
My project contains approx 3k classes and when I invoke simply enpoint I must wait about 1.5-2 seconds. This long time is due to Live reload. I checked quarkus code and I found that I can disable this functionality by Console with 'l' command.

It is not acceptable for me. I would like to have Live reload disabled by default.

Could you help and give me some hints?
Can I force launch io.quarkus.deployment.dev.RuntimeUpdatesProcessor#toggleLiveReloadEnabled in some way? 
eg with 
public void init(@Observes StartupEvent event)
or something like this? extend configuration? whatever?

Of course the best solution is allow to configure RuntimeUpdatesProcessor.liveReloadEnabled within application.properties but some workaround for now will be enough.

Thanks for any responses.
Regards :)
Michal

**Comment by mschorsch** [comment]
I am also interested in a solution to deactivate the automatic live reload via configuration.

**Comment by gsmet** [comment]
I’m on my phone so can’t check easily but IIRC we have ˋquarkus.live-reload.enabled`.

**Reply by mschorsch** [reply]
Thanks for the hint @gsmet , but I can't find the option `quarkus.live-reload.enabled` under https://quarkus.io/guides/all-config.

**Reply by mschorsch** [reply]
@gsmet The option quarkus.live-reload.enabled does not seem to exist.

**Comment by michal-duchnowski** [comment]
Any other ideas?
